src/Contexts/TodoContextProvider.tsx

```diff
@@ -0,0 +1,71 @@
+import { createContext } from 'react';
+import useSyncedReducer from '../hooks/useSyncedReducer';
+import { TodoAction } from '../Interfaces/actions';
+import { ITodoCtx, TodoList } from '../Interfaces/interface';
+
+const initialTodoList = [
```

Comment:
```suggestion
const initialTodoList: TodoList = [
```
꼼꼼히 타이핑을 해줍시다. implicit any는 지양합시당